src/pages/home/HeaderView.js

```diff
@@ -390,7 +427,7 @@ export default memo(
         },
         policy: {
             key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report ? report.policyID : '0'}`,
-            selector: (policy) => _.pick(policy, ['name', 'avatar', 'pendingAction']),
+            // selector: (policy) => _.pick(policy, ['name', 'avatar', 'pendingAction, description']),
```

Comment:
Is there a pending change here?

Reply:
Fixed here: https://github.com/Expensify/App/pull/36058/commits/50611e805b77534f663f4de5a775964c24289622